### PR TITLE
Support skip via story parameters

### LINKED ISSUE #176: No way to skip a story using CSF (Storybook 5.2)
When writing stories using the [CSF format](https://storybook.js.org/docs/formats/component-story-format/), there is no way to mark a single story to be skipped. Ideally, a parameter would be neat:

```
export const componentVariation = () => <MyComponent prop1={val1} />;
componentVariation.story = {
  parameters: { skipLoki: true },
};
```

### PR DESCRIPTION
**Pull request by oblador**
This PR moves the configuration of skipping individual stories to the story `parameter` field, deprecating the `lokiSkip` method. Additionally it drops the deprecated use of `kind+story` combination and uses the `id` introduced in SB 5. 

Fixes #176 #190 #198

**This is a breaking change**, dropping support for storybook prior to version 5.0 which was released about one year ago. It is probably possible to keep backwards support, but it would be a bit messy and would have to rely on heuristics.

### CODE REVIEW

File: examples/react/src/stories/index.stories.js [hunk 1]
```diff
@@ -17,9 +17,11 @@ import FetchComponent from '../FetchComponent';
 import ZeroHeightWithPadding from '../ZeroHeightWithPadding';
 import Hover from '../Hover';
 
-storiesOf('Welcome', module).lokiSkip('to Storybook', () => (
-  <Welcome showApp={linkTo('Button')} />
-));
+storiesOf('Welcome', module)
+  .lokiSkip('to Storybook', () => <Welcome showApp={linkTo('Button')} />)
```
Comment (techeverri) [inline examples/react/src/stories/index.stories.js:21]: So `lokiSkip` will continue to exist and behave like `add`?

File: packages/integration-core/src/decorate-storybook.js [hunk 2]
```diff
@@ -8,23 +8,23 @@ let warnedAsyncDeprecation = false;
...
-  function wrapWithSkipStory(add, kind, isDeprecatedCall) {
-    return function skipStory(story, storyFn, parameters) {
-      if (isDeprecatedCall && !warnedSkipDeprecation) {
+  function wrapWithSkipStory(add) {
+    return function skipStory(story, storyFn, parameters = {}) {
+      if (!warnedSkipDeprecation) {
         warnedSkipDeprecation = true;
         console.warn(
-          '[DEPRECATED] `.add.skip(...)` is deprecated. Please use `.lokiSkip(...)` instead.'
+          '[DEPRECATED] `.add.skip(...)` and `.lokiSkip(...)` are deprecated. Please pass `{ loki: { skip: true } }` in the third parameter instead.'
```
Comment (techeverri) [inline packages/integration-core/src/decorate-storybook.js:17]: Nice warning! 💯

File: examples/react/src/stories/index.stories.js (hunk 1 above)
Comment (oblador) [reply examples/react/src/stories/index.stories.js:21]: Yeah, i'll add warnings for now and remove it some time in the future.

File: examples/react/src/stories/csf.stories.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { linkTo } from '@storybook/addon-links';
```
Comment (techeverri) [inline examples/react/src/stories/csf.stories.js:4]: Does the example need these 2 imports? 🤔

Comment (oblador) [reply examples/react/src/stories/csf.stories.js]: Fixed!

**Comment by Tobi-mmt** [comment]
@oblador this will probably also close #103 – if we use the parameters instead of `skip()` – since the stories are filtered by there parameters without an blocking, extra function.

I am excited for this update 🥳

**Comment by johannespfeiffer** [comment]
Here too 😄 
Any eta for this update?

**Pull request merged**